Add Matrix link to socials (Hacktoberfest)

Vina07 opened the pull request:
Added a new Matrix community link for Cloud Native Mauritius: https://matrix.to/#/#cloudnativemu:matrix.meetup.mu

Vina07 committed:
Add Matrix link to socials (Hacktoberfest)

Added a new Matrix community link for Cloud Native Mauritius:
https://matrix.to/#/#cloudnativemu:matrix.meetup.mu

ish-sookun commented:
@Vina07 To keep consistency with the other icons, I suggest to use an SVG icon for Matrix. You may find one from https://simpleicons.org/?q=matrix.

Vina07 committed:
Use official SVG icon for Matrix (to keep consistency with other social icons)

Updated Matrix social icon to use the official SVG from SimpleIcons for consistent styling with other social icons.

Vina07 commented:
Updated Matrix social icon to use the official SVG from Simple Icons for consistent styling with other social icons.

xelab04 commented:
It looks like it would work. However, for the purpose of the Matrix, you put the component in the wrong place.
In `views/components/socials`, this is used for author socials for the "blogposts". Technically, this is fine actually but not exactly what I needed.

The change you made works for here:
<img width="896" height="232" alt="image" src="https://github.com/user-attachments/assets/41bdd315-3951-4ed1-9296-b25311a0c3ba" />

What we need rather is in the header:
<img width="1011" height="243" alt="image" src="https://github.com/user-attachments/assets/17181f8f-1190-463d-9f00-84841a9aa569" />

So, let me point you to resources/views/layouts. The Telegram blocks (there are two, one for desktop and one for mobile view) are on lines 43 to 50 and 71 to 78, you can use them as reference and add the Matrix block under that.

The link it would use is `https://matrix.to/#/#cloudnativemu:matrix.meetup.mu`

Vina07 committed:
Add Matrix link and icon under Telegram (desktop and mobile)

Added the Matrix community link (https://matrix.to/#/#cloudnativemu:matrix.meetup.mu) in the header navigation for both desktop and mobile layouts, right below the Telegram icon.

This follows the feedback from xelab04 to place it in resources/views/layouts/

Vina07 commented:
Integrated the Matrix link into the header (both desktop and mobile views) by adding the block under the existing Telegram elements in resources/views/layouts, and utilizing the URL https://matrix.to/#/#cloudnativemu:matrix.meetup.mu.

xelab04 commented:
Alright, looks good. Are you able to test it?

You can check the Laravel docs for this or maybe setup Herd or Sail (iirc) directly since you're on Windows.

@ishwon , any thoughts on that?

ish-sookun commented:
<img width="329" height="134" alt="pr-48" src="https://github.com/user-attachments/assets/4703007b-48db-41eb-91f7-63cb9632eab0" />

This is what I am getting when testing locally.

Did you obtain the SVG from simpleicons.org?

Vina07 committed:
Update header.blade.php

Vina07 committed:
Update matrix.blade.php

Vina07 commented:
<img width="1822" height="768" alt="Screenshot 2025-10-26 144822" src="https://github.com/user-attachments/assets/d2c4313b-2a4d-4f8b-9cc7-4c89af0a3a66" />

fix(matrix-icon): Yes I used matrix svg icon from https://simpleicons.org/?q=matrix, tested locally on my Windows machine via XAMPP (Apache, PHP) with NPM & Vite build
 using

xelab04 commented:
I checked what the logo should look like, and I think you might have to remove the rounded-full as it's cutting off the square brackets of the logo `[m]`

Vina07 commented:
<img width="1890" height="589" alt="Screenshot 2025-10-26 162455" src="https://github.com/user-attachments/assets/4705e601-b846-45e1-a755-936def8b24e1" />

Actually it looks like this, I kept the styling consistent, so the hover effect automatically inherits the existing CSS styles.

xelab04 committed:
fx: make matrix icon square

xelab04 committed:
fx: make matrix icon square

xelab04 committed:
fx: mobile matrix logo square

xelab04 committed:
Merge branch 'add-matrix-link' of https://github.com/Vina07/cloudnativemauritius.com into add-matrix-link

xelab04 committed:
fx: fix telegram logo

xelab04 commented:
I pushed some updates to your branch which make it square again. Run a git pull and check.

xelab04 committed:
fx: indentation

Vina07 commented:
<img width="1919" height="784" alt="Screenshot 2025-10-26 201524" src="https://github.com/user-attachments/assets/2f8a6cd1-05ee-4230-bee1-d2d4c0cabe9e" />

I pulled latest changes and tested everything locally on my Windows machine using XAMPP (Apache, PHP) and VS Code IDE.
The Matrix SVG icon now displays correctly with consistent hover styling.
Everything looks great, ready to merge.

xelab04 commented:
Looks fine to me

xelab04 commented:
Eyyy, I expected @ishwon to add a LGTM and then merge.
Ish, mind giving it a look before October ends?
